### PR TITLE
Add data generation script

### PR DESCRIPTION
**Pull request by PlasmaIntec**
Please review and merge @elizabethcan @pon-y

### CODE REVIEW

File: util/adapter.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+	response.data.pipe(writer);
+
+	return new Promise((resolve, reject) => {
+		writer.on('finish', resolve);
+		writer.on('error', reject);
+	});
+};
+
+// for (let i = 0; i < 1000; i++) {
```
Comment (pon-y) [inline util/adapter.js]: remove unnecessary comments if not needed. Will you be using this in the future?

File: util/pgHandler.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+		});
+
+		await makeSchema(client);
+
+		await bulkCopy(client, done);
+	});
+
+	// for (let i = 0; i < 1e6; i++) {
+	// 	await insertProperties(pool);		
+	// }
```
Comment (pon-y) [inline util/pgHandler.js]: ditto to above. Are these comments necessary?

**Review by pon-y — APPROVED** [review]

File: util/csvWriter.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+	return {
+		photo_id: PHOTO_CHUNK_SIZE * property_id + photo_id, 
+		property_id: property_id,
+		url: `https://s3.amazonaws.com/xillow-photos/${rounded_photo_id}.jpg`,
+	};
+};
+
+var generateDetail = (property_id) => {
+	return {
+		id: property_id,
```
Comment (elizabethcan) [inline util/csvWriter.js:19]: indentation is a little off here

File: util/csvWriter.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+var streamToPromise = (stream, chunk) => {
+	return new Promise((resolve, reject) => {
+		stream.write(chunk, () => {
+			resolve();
+		});
+	});
+};
+
+const writeToCsv = async (num = 1) => {
+	var propertiesStream = fs.createWriteStream('properties.csv', { flags: 'w' }),
```
Comment (elizabethcan) [inline util/csvWriter.js:51]: what does flag w do? why not flag a?

**Review by elizabethcan — APPROVED** [review]

File: util/adapter.js (hunk 1 above)
Comment (PlasmaIntec) [reply util/adapter.js]: I've already downloaded photos, will go back and delete

File: util/pgHandler.js (hunk 2 above)
Comment (PlasmaIntec) [reply util/pgHandler.js]: Nope

File: util/csvWriter.js (hunk 4 above)
Comment (PlasmaIntec) [reply util/csvWriter.js:51]: w overwrites the file if it exists and writes to empty file if it doesn't exist

I used w over a to preserve memory

File: util/csvWriter.js (hunk 3 above)
Comment (PlasmaIntec) [reply util/csvWriter.js:19]: I think this indentation better allows user to see JSON structure

**Pull request merged**